ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_service.proto

```diff
@@ -115,6 +115,9 @@ message GetTransactionsRequest {
   // Optional
   TraceContext trace_context = 1000;
 
+  // Optional
+  bool include_non_consuming = 6;
```

Comment:
```suggestion
  bool include_non_consuming_exercise_events = 6;
```

Reply:
Or just change the name to be very explicit; and thereby leaving ample space for later additions of flags.